approvaltests/reporters/generic_diff_reporter_factory.py

```diff
@@ -9,6 +9,13 @@ class GenericDiffReporterFactory(object):
 
     def __init__(self):
         self.load(get_adjacent_file('reporters.json'))
+        self.add_fallback_reporter_config(["PythonNative", "python", [get_adjacent_file("python_native_reporter.py")]])
+
+    def add_default_reporter_config(self, config):
```

Comment:
I couldn't see this being used anywhere. What was the use case for this method and the `add_fallback_reporter_config` method below?